### PR TITLE
Fixing code smell for config/repo_config.go

### PR DESCRIPTION
**Pull request by omaciel**
# Description

Fixed minor code smell issue (see screenshot) in `config/repo_config.go`.

Screenshot 2022-11-15 at 10 22 08
Screenshot 2022-11-15 at 10 21 59

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [x] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

File: config/repo_config.go [hunk 1]
```diff
@@ -23,18 +27,18 @@ var RHEL90 = []string{"ansible-core"}
...
-	"rhel-84": RHEL8,
-	"rhel-85": RHEL8,
-	"rhel-86": RHEL8X,
-	"rhel-87": RHEL8X,
-	"rhel-90": RHEL90,
+	"rhel-84":           RHEL8,
+	"rhel-85":           RHEL8,
+	"rhel-86":           RHEL8X,
+	"rhel-87":           RHEL8X,
+	DefaultDistribution: RHEL90,
```
Comment (ldjebran) [inline config/repo_config.go:34]: I am not sure about this ,  DefaultDistribution is is the pointer to default it it self 

we may have for example all the dists defined as vars but will define here only rhel-90

```go 
const distRHEL90 = "rhel-90"
const DefaultDistribution = distRHEL90

var DistributionsPackages = map[string][]string{
	...
	distRHEL90: RHEL90,
}

var DistributionsRefs = map[string]string{
	...
	distRHEL90: OstreeRefRHEL9,
}




```

**Review by ldjebran — COMMENTED** [review]
I like the idea left one comment

**Review by ldjebran — APPROVED** [review]
LGTM

**Comment by ldjebran** [comment]
ACK

**Review by acosferreira — APPROVED** [review]
lgtm

**Pull request merged**